config/markets.ts

```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
```

Comment:
i think it should be SPOT:ETH/USD

looks like we have incorrect virtualMarketId for some older synthetic markets

Reply:
agree it should be SPOT:ETH/USD